scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -536,6 +682,13 @@
                       :instance.status/running :task-running}]
     (when (seq running-tasks)
       (log/info "Preparing to reconcile" (count running-tasks) "tasks")
+      (doseq [[task-id] running-tasks
+              :let [task-ent (d/entity db task-id)
+                    hostname (:instance/hostname task-ent)
+                    job (:job/_instance task-ent)]]
+        (.. fenzo
+            (getTaskAssigner)
```

Comment:
This isn't thread safe--should be done on a channel w/ the `offer-handler`'s loop or else we'll see IllegalStateExceptions.